source/package.json

```diff
@@ -14,12 +14,7 @@
     "rest",
     "api"
   ],
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/apickli/apickli/blob/master/LICENSE"
-    }
-  ],
+  "license": "MIT",
```

Comment:
@theTestTube - out of curiosity, what tool/editor, etc are you using that gives this warning?